apps/back-end/package.json

```diff
@@ -7,8 +7,9 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "build": "vite build --ssr src/server.js --outDir dist",
+    "build": "vite build --ssr src/server.js --outDir dist && cp pm2/* dist",
     "start": "fastify start --options src/index.js",
+    "start:attached": "node src/server.js",
     "test": "c8 node --test test/test-*.js",
     "dev": "fastify start -l debug -P --options src/index.js"
```

Comment:
Note I changed this to get it working with the dev FE. I used LX's backend package.json for inspiration, which generally works well for dev/prod and we might want to copy.

Reply:
I didn't change `start` deliberately, so that the current behaviour would continue.

I don't think you're saying you changed `start:attached`? So I'm wondering what you mean...

Reply:
I changed the `dev` script. But not actually that relevant for this.